app/crud/common.py

```diff
@@ -9,6 +9,7 @@
 _TYPES = {
     "LABEL": Label,
     "PROBE": Probe,
+    "QUERY_MONITOR": Probe,
```

Comment:
"QUERY_MONITOR": Query monitor ?

Reply:
I don't want to change the entity type from being this format `ALL_CAPS` because this is critical to align between sql proc definition and the python code.

We do normalize the key by calling `lower()` on it before wrapping the error message, e.g.

```
return summarize_error(f"Failed to create {entity_type.lower()}", e)
```

We could remove the underscore if you think that's important.

Reply:
no important, ship it.